.github/workflows/autoupdate.yml

```diff
@@ -3,9 +3,12 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    types: [ labeled ]    
```

Comment:
Use this.

```
pull_request:
    types: [labeled]
    branches:
      - master
```

The branches field is present in the on.pull_request field to ensure that the pull request originates from the master branch.

Reply:
addressed